transport-udt/src/main/java/io/netty/channel/udt/nio/NioUdtByteConnectorChannel.java

```diff
@@ -150,6 +151,11 @@ protected int doWriteBytes(final ByteBuf byteBuf) throws Exception {
     }
 
     @Override
+    protected ChannelFuture shutdownInput() {
+        return voidPromise();
```

Comment:
yes I debated this but no-one is currently attaching to the future ... it only returns a ChannelFuture so it doesn't conflict with the existing `ChannelFuture shutdownInput()` from the base interface.

However I can change this in case future code changes i guess